caddy/setup/markdown.go

```diff
@@ -55,9 +34,9 @@ func markdownParse(c *Controller) ([]*markdown.Config, error) {
 
 	for c.Next() {
 		md := &markdown.Config{
-			Renderer:    blackfriday.HtmlRenderer(0, "", ""),
-			Templates:   make(map[string]string),
-			StaticFiles: make(map[string]string),
+			Renderer:   blackfriday.HtmlRenderer(0, "", ""),
+			Extensions: make(map[string]struct{}),
```

Comment:
`map[string]struct{}` is the classic Go way to make a set because `struct{}` requires 0 allocations (0 bytes in memory).